deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/data/dashboard_saturation_tests.py

```diff
@@ -0,0 +1,367 @@
+    if not isabs(args.session):
+        args.session = f'{getcwd()}/{args.session}'
+
+    # Check Artillery Result Types
+    check_artillery_result_types(args.type)
+
+    # Provide Necessary Directories
+    provide_directories(args)
+
+    # Create Session File
+    create_session_file(args.session, args.user)
+
+    # Check Artillery Parameters
+    add_options_to_artillery(args.artillery, gen_artillery_params(args))
+
+
+def gen_log_filename(log_path: str) -> str:
+    """Generate log file name.
+
+    Args:
+        log_path (str): Path of the logs.
+
+    Returns:
+        str: File name of the log (include path).
+    """
+    return log_path + datetime.now().strftime(f"log-%Y%m%d%H%M%S.log")
+
+
+def gen_url(ip: str) -> str:
+    """Generate dashboard url.
+
+    Args:
+        ip (str): IP of the dashboard.
+
+    Returns:
+        str: Complete url of the dashboard.
+    """
+    url_format = 'https://'
+
+    return f'{url_format}{ip}'
+
+
+def gen_csv_filename(csv_path: str, type: str) -> str:
+    """Generate csv file name (per type).
+
+    Args:
+        csv_path (str): Path of the CSVs.
+        type (str): CSV data type.
+
+    Returns:
+        str: File name of the csv (include type and path).
+    """
+    return csv_path + datetime.now().strftime(f"{type}-%Y%m%d%H%M%S.csv")
+
+
+def convert_json_to_csv(args: argparse, json_output: str) -> None:
+    """Convert data from JSON format to CSV format.
+
+    Args:
+        args (argparse): Script parameters.
+        json_output (str): Path and file name of the log.
+    """
+    for type in args.type:
+        with open(json_output) as f:
+            data = json.load(f)
+
+        csv_filename = gen_csv_filename(args.csv, type)
+
+        df = pd.json_normalize(data[type])
+        df.to_csv(csv_filename, index=False)
+
+
+def run_artillery(args: argparse) -> None:
+    """Execute Artillery tests.
+
+    Args:
+        args (argparse): Script parameters.
+    """
+    json_filename = gen_log_filename(args.logs)
+
+    params = f"-v '{json.dumps(gen_artillery_params(args))}'"
+    target = f"-t {gen_url(args.ip)}"
+
+    # Enable Quiet Mode (Artillery)
+    quiet = ''
+
+    if not args.debug:
+        quiet = f'-q'
+
+    output = f"-o {json_filename}"
+    script = f"{args.artillery}"
+
+    command = f'artillery run {params} {target} {quiet} {output} {script}'
+
+    run(command, shell=True)
+    convert_json_to_csv(args, json_filename)
+
+
+def get_script_arguments() -> argparse:
+    """Add and Receive the Script Parameters.
+
+    Returns:
+        argparse: Script parameters.
+    """
+
+    parser = argparse.ArgumentParser(
+        usage='%(prog)s [options]',
+        description='Script to Run Dashboard Saturation Tests',
+        formatter_class=argparse.RawTextHelpFormatter
+    )
+
+    parser.add_argument(
+        '-l', '--log',
+        dest='logs',
+        default='logs/',
+        help=f'Directory to store the logs. Default "logs".'
+    )
```

Comment:
I think you should include the type of parameter (`str`) since you have put it in some of them, it should appear in all of them.

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5629/commits/7b6d04e9738b6ecfa7b067089d145b2ee5dd4603